pack/mts_encounter.json

```diff
@@ -1283,7 +1285,7 @@
 		"scheme_star": true,
 		"set_code": "infinity_gauntlet",
 		"set_position": 1,
-		"text": "Permanent. setup\n[star] <b>Forced Response</b>: After attached villain activates, resolve the <b> Special </b> ability of each [[infinity stone]] in play. Otherwise, put the top card of the [[infinity stone]] deck into play.",
+		"text": "Permanent. setup\n[star] <b>Forced Response</b>: After attached villain activates against you, resolve the <b> Special </b> ability of each [[infinity stone]] in play. Otherwise, put the top card of the [[infinity stone]] deck into play.",
```

Comment:
Existing issue but I think this should say `Permanent. Setup.` instead of `Permanent. setup.`